optaplanner-quarkus-integration/optaplanner-quarkus/runtime/pom.xml

```diff
@@ -110,6 +110,17 @@
           </annotationProcessorPaths>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <!-- TODO enable dependency check later. -->
+            <id>analyze-only</id>
+            <phase>none</phase>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
There's a skip option:
  https://maven.apache.org/plugins/maven-dependency-plugin/analyze-only-mojo.html#skip

Can we not add a property


```
<properties>
<mdep.analyze.skip>true</>
</>
```

for all these kind of cases? It's less lines of code :)

Reply:
No. Using the property has a different effect. It disables the plugin execution for the `analyze` goal completely. With that, you couldn't run analysis from the command line.